src/components/media/slideshow/SlideViewer.tsx

```diff
@@ -162,6 +168,12 @@ const RhsContainer = ({ loaded, userinfo, content, footer = null }: RhsContainer
             {content}
           </div>
         </div>
+        {keyboardTip &&
+          <div className='mb-2'>
+            <span className='text-xs px-4 text-slate-500'>
+              Keyboard shortcuts: Previous [<kbd className='kbd-xs'>◀︎</kbd>] Next [<kbd className='kbd-xs'>▶︎</kbd>]
+            </span>
+          </div>}
```

Comment:
May I request a small change to simplify things a bit?
1.   I'd move the showing/hidding keyboardTip to InfoContainer component. 
3.   Introduce `keyboardTip` prop to InfoContainer and make it optional, default = true

```javascript
interface InfoContainerProps {
  currentImage: MediaType | null
  tagList: HybridMediaTag[]
  auth: WithPermission
  keyboardTip?: boolean
  onClose?: () => void
}

// set keyboardTip to true if omitted 
const InfoContainer = ({ currentImage, tagList, auth, keyboardTip = true, onClose }: InfoContainerProps): ReactElement | null => {
...
```
4.  In SingleViewer component (which is used by /p/[...slug].tsx page) pass `keyboardTip={false}` prop to InfoContainer.

Other than that you don't need introduce keyboardTip to other parent components.  Let me know if you have any questions.

Reply:
Thanks for the feedback, that is a much cleaner approach.

Reply:
can you remove unused `keyboardTip` prop in UserGallery, p/[..slug] and /u/[...slug]?  We're good to merge then.

Reply:
thanks for cleaning that up for me and merging.

Reply:
@tedgeving no I haven't cleaned it up yet.  Can you please do it and push again?
 I guess you got confused by the all-contributors bot notification (a separate PR to add your name to the list of contributors).

Reply:
I tried to push a small update to your PR but it didn't go through for some reason.  I created a new one based on this #677.
Thanks for your help!